collectors/apps.plugin/apps_plugin.c

```diff
@@ -2191,7 +2193,11 @@ static inline int read_pid_file_descriptors(struct pid_stat *p, void *ptr) {
             p->fds[fdid].cache_iterations_counter = p->fds[fdid].cache_iterations_reset;
         }
     }
-
+    usedfdperc = (p->fds_size * 100) / get_system_fd_max();
```

Comment:
Reading this part of your code, it looks like you need to rename the variables. If something is bigger than the `max`, should I really adjust the max? Before to reach this part of the code, I understood that the maxfdperc, should be a boundary value. :thinking:

Reply:
I will look into this, I may rename it to `currentmax...` or something similar. `usedfdperc` is the used file descriptors percentage. As mentioned in #3202 we should get the max value of the file descriptors used so `maxfdperc` is the max percentage that I gathered from the PIDs and then I shall pass that into the alarm. Will edit the names to better display the meaning of each variable!